org.eclipse.scanning.points/src/org/eclipse/scanning/points/serialization/PositionSerializer.java

```diff
@@ -27,7 +27,7 @@
 	public void serialize(IPosition pos, JsonGenerator gen, SerializerProvider prov) throws IOException, JsonProcessingException {
 
 		try {
-			final PositionBean bean = new PositionBean(pos);
+			final PositionBean<?> bean = new PositionBean<>(pos);
 			gen.writeObject(bean);
 		} catch (Throwable ne) {
 			ne.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1148)